src/processing/reads.jl

```diff
@@ -109,31 +100,44 @@ function process_exclusions(
     else
         if isfile(exclusion_tif_path)
             exclusion_zones_int = Raster(exclusion_tif_path, mappedcrs=EPSG(EPSG_code), missingval=0)
-            exclusion_zones_bool = Raster(exclusion_zones_int .!= 0, dims(exclusion_zones_int))
+            exclusion_zones_bool = exclusion_zones_int .!= 0
```

Comment:
I'm not really a fan of adding the types in variable names, I really only do it for examples or if tracking the types is really necessary for debugging.

Here, the types indicated aren't even correct (`exclusion_zones_bool` does not hold a boolean).

A more informative variable name or a comment is better in this case.